tsconfig.json

```diff
@@ -17,7 +16,10 @@
     "moduleResolution": "node",
     "skipLibCheck": true,
     "tsBuildInfoFile": ".tsbuildinfo",
-    "lib": ["es6", "dom", "es2017"]
+    "lib": ["es6", "dom", "es2017"],
+    "incremental": true,
```

Comment:
This is such a small but great improvement! 👍

Reply:
Could I ask what these changes are doing? I'm not very familiar with tsconfig options.

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#faster-subsequent-builds-with-the---incremental-flag

It makes things faster.

Reply:
Ahh ok, it was already in the config, just moved to this line. Does the placement in the config make a difference?

Reply:
The placement does not make a difference here.